code-scanning/semgrep.yml

```
@@ -28,7 +28,7 @@ jobs:
       - uses: actions/checkout@v2
 
       # Scan code using project's configuration on https://semgrep.dev/manage
-      - uses: returntocorp/semgrep-action@fcd5ab7459e8d91cb1777481980d1b18b4fc6735
+      - uses: returntocorp/semgrep-action@v1
```

I'm afraid we still need the full 40 character SHA, but pointing to the appropriate 'v1' commit... 🙂 
```suggestion
      - uses: returntocorp/semgrep-action@b4ae418326a5e8bd4fc3b0b658695aee09ca0e2a
```

---

@DrewDennison just in case you missed the ping